src/vtt-segment-loader.js

```diff
@@ -476,11 +476,13 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     const timestampmap = segmentInfo.timestampmap;
     const diff = (timestampmap.MPEGTS / ONE_SECOND_IN_TS) - timestampmap.LOCAL + mappingObj.mapping;
+    const mpegTsRollover = Math.floor((2 ** 33) / ONE_SECOND_IN_TS);
```

Comment:
Since this code isn't isolated to TS segments, we may want to use the actual media timescale rather than `ONE_SECOND_IN_TS` (90000).